3.extract_features/pipelines/OP_pipe.cppipe

```diff
@@ -0,0 +1,199 @@
+    Relative pixel spacing in X:1.0
+    Relative pixel spacing in Y:1.0
+    Relative pixel spacing in Z:1.0
+    Select the rule criteria:and (file does endwith ".tiff")
+    Name to assign these images:OP
+    Name to assign these objects:Cell
+    Select the image type:Grayscale image
+    Set intensity range from:Image metadata
+    Maximum intensity:255.0
+    Select the rule criteria:and (file does endwith ".png")
+    Name to assign these images:Mask
+    Name to assign these objects:Mask
+    Select the image type:Binary mask
+    Set intensity range from:Image metadata
+    Maximum intensity:255.0
+
+Groups:[module_num:4|svn_version:'Unknown'|variable_revision_number:2|show_window:False|notes:['Group by plate to run independently of each other.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Do you want to group your images?:No
+    grouping metadata count:1
+    Metadata category:Plate
+
+ConvertImageToObjects:[module_num:5|svn_version:'Unknown'|variable_revision_number:1|show_window:False|notes:[]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Select the input image:Mask
+    Name the output object:ConvertImageToObjects
+    Convert to boolean image:Yes
+    Preserve original labels:No
+    Background label:0
+    Connectivity:0
+
+MeasureGranularity:[module_num:6|svn_version:'Unknown'|variable_revision_number:4|show_window:False|notes:['This module measures image granularity by default and we have the parameter "measure within objects" turned on so that object measurements for granularity are measured.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Select images to measure:OP
+    Measure within objects?:Yes
+    Select objects to measure:ConvertImageToObjects
+    Subsampling factor for granularity measurements:0.25
+    Subsampling factor for background reduction:0.25
+    Radius of structuring element:10
+    Range of the granular spectrum:16
+
+MeasureObjectIntensity:[module_num:7|svn_version:'Unknown'|variable_revision_number:4|show_window:False|notes:['This module measures the intensity within each individual object']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Select images to measure:OP
+    Select objects to measure:ConvertImageToObjects
+
+MeasureImageIntensity:[module_num:8|svn_version:'Unknown'|variable_revision_number:4|show_window:False|notes:['This is set to measure intensity for the whole image.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Select images to measure:OP
+    Measure the intensity only from areas enclosed by objects?:No
+    Select input object sets:Cells, Cytoplasm, Nuclei
+    Calculate custom percentiles:No
+    Specify percentiles to measure:10,90
+
+MeasureObjectNeighbors:[module_num:9|svn_version:'Unknown'|variable_revision_number:3|show_window:False|notes:['This measures adjacent nuclei neighbors.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```

Comment:
Recommend changing `nuclei` to `bones` or whatever you call the objects in this note from the GUI.